recipes/signxml/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "signxml" %}
+{% set version = "2.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "70e3edbb07b89bec94d39db2cdced724c540f9258366474177c746b9f903d9c4"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
why dropping `--no-deps`? Without changing requirements will not be catched by the import tests.

Reply:
@marcelotrevisani Why dropping --no-deps? Without changing requirements will not be catched by the import tests.

Reply:
``conda-build`` already handles it

Reply:
okay, thanks!